src/cli/commands/fellowship/endTurn.ts

```diff
@@ -1,13 +1,13 @@
 import { Command } from "commander";
 import chalk from "chalk";
 import ora from "ora";
-import { getAddress } from "viem";
 import { createPublic, createWallet } from "../../client";
 import GameMaster from "../../../rankify/GameMaster";
+import { CLIUtils } from "../../utils";
 
 export const endTurn = new Command("endTurn")
   .description("End turn in a game")
-  .argument("<instance>", "Address of the Rankify instance")
+  .argument("<instance>", "Address or instanceId of the Rankify instance")
```

Comment:
this is highly confusing. I would prefer separate params. 

Also, you might have separate command that does instance deployment + new game in one go, assuming that's reason why you added this